### ISSUE TITLE
Capture route automatically

### ISSUE THREAD

**Issue by googie0815** [umap-project/umap#1360]
**Is your feature request related to a problem? Please describe.**
It would be helpful if a route on a road between two points could be captured automatically. 

**Describe the solution you'd like**
Routes on roads currently have to be traced at great expense. It would be great if it "traced" the route itself. Hard to explain what I mean

**Additional context**
Before: ![1](https://github.com/umap-project/umap/assets/56229424/0d8a2f94-39ef-4acb-9f38-b9f530d14746)
After: ![2](https://github.com/umap-project/umap/assets/56229424/f470cafa-e1ca-4072-89bb-ab3c5b39cd35)

**Comment by yohanboniface** [comment]
Yes, what you want is routing. See #297 

It's indeed a nice to have! But this needs an external service (like OSRM or Graphopper), so usage limitations, etc.

We could use http://www.liedman.net/leaflet-routing-machine/ for the frontend part, and have an opt-in/configuration per uMap instance. But this still needs time to be implemented.

I'm closing in favor of #297 but the needs is indeed valid!
